test/sampledata/dataspace/bcrimport/run_log.csv

```diff
@@ -0,0 +1,3 @@
+run_number,run_application,run_information
+20230202_igblast,,
+20230202_vquest,,
```

Comment:
do we anticipate more test data, or is that plan to wait for this to bake before we need it for automation etc.

Reply:
Yes, i had added sample files in my first commit, then unintentionally committed the delete of the files as i was testing something locally. Now they are added again.